src/app/speakers/[slug]/page.js

```diff
@@ -0,0 +1,80 @@
+import { speakers2024 } from '@/data/data';
+import Image from 'next/image';
+import { FaLinkedinIn, FaRegEnvelope } from 'react-icons/fa';
+import { CiGlobe, CiLinkedin } from 'react-icons/ci';
+import { IoLogoLinkedin } from 'react-icons/io5';
+import TitleWithSubtitle from '@/components/elements/TitleWithSubtitle';
+
+
+const SpeakerDetails = ({ params }) => {
+
+  const speaker = speakers2024.find(s => s.slug === params.slug);
+
+
+  if (!speaker) {
+    throw new Error('Speaker not found'); // This will trigger the error.js component
+  }
+  return (
+    <div className="flex flex-col  max-w-7xl items-start mx-auto justify-center p-8 gap-8 my-10 md:my-24">
+
+      <div className="w-full flex flex-col text-center justify-center gap-1">
+        <h2
+          className="text-[min(12vw,60px)] leading-[1.3] tracking-tighter font-semibold text-transparent bg-clip-text bg-gradient-to-b from-black to-gray-700 mb-1">{speaker.name}</h2>
+        <p className="text-gray-600 mb-4">{speaker.title}</p>
```

Comment:
I would remove this for now until we properly implement the session info.

Reply:
```

          {/* Removed till added session completely */}
          {/*<h2*/}
          {/*  className="text-[min(12vw,35px)] leading-[1.3] tracking-tighter font-semibold text-transparent bg-clip-text bg-gradient-to-b from-black to-gray-700 mb-4">{speaker.headline}</h2>*/}
          
```